src/util/prometheus/Label.cpp

```diff
@@ -73,7 +60,7 @@ Label::serialize() const
 
 Labels::Labels(std::vector<Label> labels) : labels_(std::move(labels))
 {
-    std::sort(labels_.begin(), labels_.end());  // NOLINT
+    std::ranges::sort(labels_);
```

Comment:
@kuznetsss please check that the changes to support this change don't break anything (tests pass fine). the code is equivalent as far as i can tell.

Reply:
Looks good.